build-tools-internal/src/main/groovy/elasticsearch.ide.gradle

```diff
@@ -137,15 +137,15 @@ if (providers.systemProperty('idea.active').getOrNull() == 'true') {
     }
   }
 
-  // modifies the idea module config to enable preview features on 'elasticsearch-native' module
+  // modifies the idea module config to enable preview features on ':libs:native' module
   tasks.register("enablePreviewFeatures") {
     group = 'ide'
     description = 'Enables preview features on native library module'
     dependsOn tasks.named("enableExternalConfiguration")
 
     doLast {
       ['main', 'test'].each { sourceSet ->
-        modifyXml(".idea/modules/libs/native/elasticsearch.libs.elasticsearch-native.${sourceSet}.iml") { xml ->
+        modifyXml(".idea/modules/libs/native/elasticsearch.libs.${project.project(':libs:native').name}.${sourceSet}.iml") { xml ->
```

Comment:
can't we just hard code the project name here instead of referencing another projects project instance?

Reply:
Yes, the the point here is so that if that project name changes, this will explicitly fail. A missing file is leniently ignored.